.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Deployment
+
+on:
+  release:
+    types: [published]
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Ballerina Build
+              uses: ballerina-platform/ballerina-action/@master
+              with:
+                  args:
+                      build -c --skip-tests ./simpledb
```

Comment:
Need to change according to latest ballerina version

Reply:
Done